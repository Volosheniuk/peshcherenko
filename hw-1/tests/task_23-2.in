a
23
3.141592
3.14159264
