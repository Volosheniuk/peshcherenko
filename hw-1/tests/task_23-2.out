a
23
3.141592
3.141593
